app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRouter.kt

```diff
@@ -43,14 +44,24 @@ fun ConversationRouterHomeBridge(
 ) {
     val conversationState = rememberConversationState()
     val viewModel: ConversationListViewModel = hiltViewModel()
+    val mutingConversationState = rememberMutingConversationState(conversationState.modalBottomSheetContentState.value.mutedStatus)
```

Comment:
I wonder if we really need another state object living seperated from ConversationState object, perhaps we could merge it together with ConversationState, we could have a flag inside ConversationState showMuteSettings or something like that

Reply:
I tried to have this new sheet the most isolated possible, because we will have to reuse it (without the conversations' menu options) later on, specifically in Other user profile and conversation details.

So, I think I'll leave as it is right now, since having a separated state will allow us to achieve that later on.

Reply:
makes sense :) 👍